src/lib/bootloader/udev_mapping.rb

```diff
@@ -58,8 +58,12 @@ def staging
     end
 
     def udev_to_kernel(dev)
+      dev_path = DevicePath.new(dev)
+
       # in mode config if not found, then return itself
       return dev if Yast::Mode.config
+      # in installation do not care of uuids - not know at this time
+      return dev if Yast::Mode.installation && dev_path.uuid?
```

Comment:
this sadly does not work as expected, as udev mapping is called also in inst_finish, where partitioning and all uuid is known and needs to be handled properly.

Reply:
well, it worked as expected, bud i didn't know that it is effective even in the finish script ;-)